MFBNavigationTests/MFBPushPopNavigatorSpec.m

```diff
@@ -683,6 +685,14 @@
                                           animated:YES];
         });
 
+        it(@"doesn't suspend queue on willShowViewController when animated NO", ^{
```

Comment:
`when not animated`

Reply:
Also, I'm confused a bit, why is this test in `describe(@"delegate forwarding"` part of tests?

Reply:
Silly me, now I see.

Reply:
Actually, I think tests for this change deserve their own section in the spec. Like "external navigation" or similar.